src/postgres/models/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
what is the difference between this file and src/models/index?

Reply:
this I generated with sequelize-cli, src/models/index was previously written by me